aws-s3-private-bucket/main.tf

```diff
@@ -1,6 +1,8 @@
 locals {
   # If grants are defined, we use `grant` to grant permissions, otherwise it will use the `acl` to grant permissions
-  acl = length(var.grants) == 0 ? "private" : null
+  acl = length(var.grants) == 0 ? "private" : (
+    var.log_delivery_write_acl_enable ? "log-delivery-write" : null
```

Comment:
Think this needs a fix -- the log delivery flag will only work here if we have grants AND log_delivery_write_acl_enable set.

Reply:
redid, but had to put in a try because https://github.com/hashicorp/terraform/issues/25014

Reply:
I did find this: https://github.com/hashicorp/terraform/issues/22131#issuecomment-513020615, which is what I had previously.